components/post_view/post/post.jsx

```diff
@@ -13,7 +13,7 @@ import * as PostUtils from 'utils/post_utils.jsx';
 import * as Utils from 'utils/utils.jsx';
 
 import PostBody from 'components/post_view/post_body';
-import PostHeader from 'components/post_view/post_header';
+import PostHeader from 'components/post_view/post_header/post_header.jsx';
```

Comment:
If we're not using the container, can we remove it and delete the post_header folder? That way this can still be `components/post_view/post_header` and we don't have a container we're not using lying around

Reply:
Ya ya, i was gonna ask about that in pre-release before deleting.